x-pack/solutions/security/plugins/cloud_security_posture/public/pages/configurations/findings_flyout/findings_flyout.tsx

```diff
@@ -168,7 +170,7 @@ const getFlyoutDescriptionList = (finding: CspFinding): EuiDescriptionListProps[
   ].filter(truthy);
 
 const FindingsTab = ({ tab, finding }: { finding: CspFinding; tab: FindingsTab }) => {
-  const { application } = useKibana().services;
+  const { application } = useKibana<CoreStart & CspClientPluginStartDeps>().services;
```

Comment:
We have been using this combined type in a quite few places now, I think it's time to add a dedicate type? 

i.e 

```typescript
export type CspUseKibanaServices = CoreStart & CspClientPluginStartDeps
```

Reply:
hmmm I remembered we did that but we revert to combined type maybe I remembered it wrongly ?
There's a reason why we did this in the first place